ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -44,6 +48,47 @@
     footer = "EthSigner is licensed under the Apache License 2.0")
 public class EthSignerBaseCommand implements Config {
 
+  static class TlsServerOptions implements TlsOptions {
+
+    @Option(
+        names = "--tls-keystore-file",
+        description =
+            "Path to PKCS#12 format trust store for the JSON-RPC HTTP service client trust.",
+        arity = "1",
+        required = true)
+    private File keystoreFile;
+
+    @Option(
+        names = "--tls-keystore-password-file",
+        description = "Path to a file containing the password required to use the truststore file.",
+        arity = "1",
+        required = true)
+    private File keystorePasswordFile;
+
+    @Option(
+        names = "--tls-client-whitelist-file",
+        description =
+            "Path to a file containing the fingerprints of authorised clients. "
+                + "Any client may connect if not specified.",
```

Comment:
```suggestion
                + "If whitelist is not specified, any client may connect.",
```

Reply:
In other words, "to enable client authentication, the whitelist file is required"

Reply:
```suggestion
                + "Any client may connect if this option is not specified.",
```